spec/jobs/distribution_reminder_mailer_job_spec.rb

```diff
@@ -21,4 +21,48 @@
       end
     end
   end
+
+  describe "conditionally sending the emails" do
+    let(:organization) { create :organization }
+
+    let(:past_distribution) { create(:distribution, issued_at: Time.zone.now - 1.week) }
+    let(:future_distribution) { create(:distribution, issued_at: Time.zone.now + 1.week) }
+
+    let(:partner_with_no_reminders) { create :partner, send_reminders: false }
+    let(:partner_with_reminders) { create :partner, send_reminders: true }
+    let(:distribution_without_reminder) { create(:distribution, partner: partner_with_no_reminders) }
```

Comment:
Do you think it's worth it to explicitly set the `issued_at` in the future for these here?

Reply:
That's a great idea!